WowPacketParserModule.V4_4_0_54481/Parsers/ChatHandler.cs

```diff
@@ -40,7 +40,7 @@ public static void HandleServerChatMessage(Packet packet)
             packet.ReadInt32("AchievementID");
 
             uint chatFlags = 0;
-            if (ClientVersion.AddedInVersion(ClientVersionBuild.V4_4_1_57294))
```

Comment:
@Ovahlord this is the change i talked about. Scan through this pr pls if it makes your life in any way harder, if it does, ill have to duplicate some things massively but it is doable.